### PR TITLE
create Contact List based on data

### PR DESCRIPTION
**Pull request by Yana-Kalandarova**

### CODE REVIEW

File: src/components/ContactList.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ContactItem from './ContactItem';
+
+const ContactList = props => (
+  <ul className="contact_list">
+    {
+      props.list.map(el => <ContactItem contact={el} key={el.name} />)
```
Comment (maximshenets) [inline src/components/ContactList.js]: You should add id field for each contact and use it as key

File: src/data.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+export default [
+  {
+    name: 'Contact 0',
+    phoneNumber: '+37529xxxxx00',
```
Comment (maximshenets) [inline src/data.js]: xxx is not needed, cuz you will be add contact as a number so full phone number should be displayed

File: src/components/ContactInfo.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const ContactInfo = props => (
```
Comment (maximshenets) [inline src/components/ContactInfo.js]: not necessary component this could be done in ContactItem

Comment (Yana-Kalandarova) [reply src/components/ContactInfo.js]: Moved to ContactItem.

File: src/components/ContactList.js (hunk 1 above)
Comment (Yana-Kalandarova) [reply src/components/ContactList.js]: Added id field to each contact and use it as a key.

File: src/data.js (hunk 2 above)
Comment (Yana-Kalandarova) [reply src/data.js]: Replaced xxx with digits

File: src/components/ContactItem.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const props = {
```
Comment (maximshenets) [inline src/components/ContactItem.js]: Usually for it is `propTypes`

Comment (Yana-Kalandarova) [reply src/components/ContactItem.js]: Done.

**Pull request merged**